upload/admin/controller/extension/module/smaily_for_opencart.php

```diff
@@ -440,7 +453,78 @@ public function index() {
         } else {
             $data['rss_limit'] = $this->config->get('module_smaily_for_opencart_rss_limit');
         }
+        // Abandoned Cart status table.
+        $this->load->model('extension/smailyforopencart/admin');
+        $url_parameters = array();
+        if (isset($this->session->data['user_token'])) {
+            $url_parameters['user_token'] = $this->session->data['user_token'];
+        }
+
+        if (isset($this->request->get['sort'])) {
+            $sort = $this->request->get['sort'];
+        } else {
+            $sort = 'customer_id';
+        }
 
+        if (isset($this->request->get['order'])) {
+            $order = $this->request->get['order'];
+        } else {
+            $order = 'ASC';
+        }
+
+        if (isset($this->request->get['page'])) {
+            $page = $this->request->get['page'];
+        } else {
+            $page = 1;
+        }
+
+        if ($order === 'ASC') {
+            $url_parameters['order'] = 'DESC';
+        } else {
+            $url_parameters['order'] = 'ASC';
+        }
+
+        if (isset($this->request->get['page'])) {
+            $url_parameters['page'] = $this->request->get['page'];
+        }
```

Comment:
You can simplify this a bit:

```suggestion
        if (isset($this->request->get['page'])) {
            $page = $this->request->get['page'];
            $url_parameters['page'] = $page;
        } else {
            $page = 1;
        }

        if ($order === 'ASC') {
            $url_parameters['order'] = 'DESC';
        } else {
            $url_parameters['order'] = 'ASC';
        }
```